osu.Game.Rulesets.Osu/Difficulty/OsuLegacyScoreSimulator.cs

```diff
@@ -133,22 +129,25 @@ private void simulateHit(HitObject hitObject)
                     // The spinner object applies a lenience because gameplay mechanics differ from osu-stable.
                     // We'll redo the calculations to match osu-stable here...
                     const double maximum_rotations_per_second = 477.0 / 60;
-                    double minimumRotationsPerSecond = IBeatmapDifficultyInfo.DifficultyRange(playableBeatmap.Difficulty.OverallDifficulty, 3, 5, 7.5);
+
+                    // Normally, this value depends on the final overall difficulty. For simplicity, we'll only consider the worst case that maximises bonus score.
```

Comment:
If anything I'd maybe consider expanding this with an explanation of _why_ we're maximising bonus score. Something like

```suggestion
                    // Normally, this value depends on the final overall difficulty. For simplicity, we'll only consider the worst case that maximises bonus score.
                    // As we're primarily concerned with computing the maximum theoretical final score,
                    // this will have the final effect of slightly underestimating bonus score achieved on stable when converting from score V1.
```

perhaps?